extensions/stats/plugin.cc

```diff
@@ -623,8 +619,7 @@ bool PluginRootContext::report(::Wasm::Common::RequestInfo& request_info,
   }
 
   incrementMetric(cache_misses_, 1);
-  // TODO: When we have c++17, convert to try_emplace.
-  metrics_.emplace(istio_dimensions_, stats);
+  metrics_.try_emplace(istio_dimensions_, stats);
```

Comment:
cool.. we should do this for all of our code in a separate PR perhaps...

Reply:
Agree. Started with Wasm sources since every cycle counts there.